src/status.h

```diff
@@ -57,13 +57,18 @@ class Status {
     NetSendErr,
   };
 
-  Status() : Status(cOK, "ok") {}
-  explicit Status(Code code, std::string msg = "") : code_(code), msg_(std::move(msg)) {}
+  Status() : Status(cOK) {}
+  explicit Status(Code code, std::string msg = {}) : code_(code), msg_(std::move(msg)) {}
   bool IsOK() { return code_ == cOK; }
   bool IsNotFound() { return code_ == NotFound; }
   bool IsImorting() { return code_ == SlotImport; }
```

Comment:
```suggestion
  bool IsImporting() { return code_ == SlotImport; }
```

Typo? May we fix it in another PR though.

Reply:
it's typo

Reply:
Maybe open a help wanted issue to encourage more involvement🤣 
Not sure whether it is a proper way, calling expert @tisonkun

Reply:
🤣

Reply:
There are several typos in our codebase. I don't think they deserve an issue but if you'd like to fix it with another patch, go ahead. Otherwise I may handle it later this week =。=

Reply:
Got it. Thanks.